jupyterlab_latex/__init__.py

```diff
@@ -8,7 +10,18 @@
 
 path_regex = r'(?P<path>(?:(?:/[^/]+)+|/?))'
 
-def _jupyter_server_extension_paths():
+HERE = Path(__file__).parent.resolve()
```

Comment:
as we do almost the same thing in `_version`, i usually hoist the full `package.json` as e.g. `__js__` and import it here.

Reply:
Great idea, DRY